src/frontend/src/pages/CreateChangeRequestPage/CreateProposedSolutionsList.tsx

```diff
@@ -51,11 +64,32 @@ const CreateProposedSolutionsList: React.FC<CreateProposedSolutionsListProps> =
             proposedSolution={proposedSolution}
             onDelete={removeProposedSolution}
             showDeleteButton
+            onEdit={(proposedSolution) => {
+              setEditingProposedSolution(proposedSolution);
+            }}
           />
         ))}
       </div>
-
-      <ProposedSolutionForm onAdd={addProposedSolution} open={showEditableForm} onClose={() => setShowEditableForm(false)} />
+      {showCreateForm && (
+        <ProposedSolutionForm
+          onSubmit={addProposedSolution}
+          open={showCreateForm}
+          onClose={() => setShowCreateForm(false)}
+        />
+      )}
+      {showEditForm && (
+        <ProposedSolutionForm
+          isEditing
+          onSubmit={editProposedSolution}
+          open={showEditForm}
+          onClose={() => setEditingProposedSolution(undefined)}
+          description={editingProposedSolution?.description}
+          budgetImpact={editingProposedSolution?.budgetImpact}
+          timelineImpact={editingProposedSolution?.timelineImpact}
+          scopeImpact={editingProposedSolution?.scopeImpact}
+          id={editingProposedSolution?.id}
```

Comment:
you should just pass in the whole editingProposedSolution as a optional "defaultValues" prop. It being defined/ undefined can also act as the isEditing prop